assets/app/styles/_core.less

```diff
@@ -483,8 +483,13 @@ label.checkbox {
   margin-left: 3px;
 }
 
-// Misc
+
+// Misc 
+// TODO: pull out into _partials where it makes sense 
 // --------------------------------------------------
+a[ng-click] {
+  cursor: pointer;
+}
```

Comment:
The cursor will be a pointer already if the `a` element has an `href`, and the style shouldn't be needed.